config/locales/en.yml

```diff
@@ -1,3 +1,24 @@
+# Files in the config/locales directory are used for internationalization
```

Comment:
we can keep this removed of you don't mind

Reply:
✅